crates/node-db-sql/src/lib.rs

```diff
@@ -109,26 +109,27 @@ pub mod table {
     }
 
     pub const BLOCK: Table = Table::new("block", create::BLOCK);
-    pub const BLOCK_SOLUTION: Table = Table::new("block_solution", create::BLOCK_SOLUTION);
-    pub const DEC_VAR: Table = Table::new("dec_var", create::DEC_VAR);
+    pub const BLOCK_SOLUTION_SET: Table =
+        Table::new("block_solution_set", create::BLOCK_SOLUTION_SET);
+    pub const PRED_DATA: Table = Table::new("pred_data", create::PRED_DATA);
```

Comment:
if we want to keep these in alphabetical order it looks like we might wanna move this const too